src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -45,6 +52,7 @@ const firestoreUpdate = (doc: string, update: any) => {
 };
 
 const defaultTables = [...Array(TABLES)].map((_, i: number) => createTable(i));
+const ALLOWED_EMPTY_TABLES_NUMBER = 4;
```

Comment:
This should be defined in `src/settings`

Reply:
fixed in 628728c978f8946518261612288a3324163ca176